update-files.py

```diff
@@ -0,0 +1,52 @@
+import re
+from git import Repo
+import subprocess
+from fic_modules.configuration import LOGGER
+from datetime import datetime
+
+
+def git_add(files, LOGGER, config="./"):
+    try:
+        repo = Repo(config)
+        for file in files:
+            repo.git.add(file, update=True)
+            if not repo.index.diff("HEAD"):
+                LOGGER.info("nothing staged for commit. has the data or files changed?")
+    except:
+        LOGGER.info("Failed to add the files")
+
+
+def git_commit(msg, LOGGER, config="."):
+    LOGGER.info("Committing changes with message: %s", msg)
+    repo = Repo(config)
+    commit = repo.index.commit(msg)
+    for patch in repo.commit("HEAD~1").diff(commit, create_patch=True):
+        LOGGER.info(patch)
+
+
+def git_push(LOGGER, config="."):
+    try:
+        repo = Repo(config)
+        LOGGER.info("pushing changes to %s", repo)
+        push_info = repo.remotes.origin.push(refspec='issue-366')
```

Comment:
This needs to be changed to "master" before we merge it and we add it to this week's release.